### DISCUSSION TITLE
I LOVE IT

### DISCUSSION THREAD

**Discussion by morgan-256** [ramensoftware/windhawk-mods#1] [General]
Just want to say WOOHOO and THANKS! I was totally addicted to separating taskbar icons with 7++ - thank you for making the mod to do it again in Win11!! YAY!!!!!

**Comment by m417z** [comment]
Awesome, thanks for the feedback! I'm glad that it works for you. More taskbar-related mods are planned for Windows 11, stay tuned.

**Reply by jeremiahbenon6-dev** [reply]
WHY DID THIS APP REFUSE TO OPERATE IN MY PC

**Reply by m417z** [reply]
Please provide more details. Are you getting an error message?

**Comment by Alpakash** [comment]
Also just wanted to say thank you for creating windhawk-mods. You're a hero, love it to have labels shown again into the taskbar in Windows 11. Cheers! 🍻
